libs/form-component/src/styles/global.css

```
@@ -0,0 +1,178 @@
+@import url('https://altinncdn.no/fonts/altinn-din/altinn-din.css');
```

Stylelint (17.11.0)

[error] Expected "url('https://altinncdn.no/fonts/altinn-din/altinn-din.css')" to be "'https://altinncdn.no/fonts/altinn-din/altinn-din.css'" (import-notation)

(import-notation)